Portfolio.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/myStyle.css"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+  </head>
+
+
+  <body>
+    <main class="content">
+      <header class="header-container">
+          <div class="Logo">
+            <img src="Logo.png" alt="log"/>
+          </div>
+
+          <div class="hamburger-btn">
+            <i class="fa fa-bars"></i>
+          </div>
+      </header>
```

Comment:
- [ ] The spacing provided for the header section is a bit inaccurate as compared to the spacing for the design. Please improve(increase) the margin around the header section.